swe_cp.m

```diff
@@ -1088,30 +1089,30 @@ function swe_cp(SwE)
     fprintf('%s%30s',repmat(sprintf('\b'),1,30),'...saving results'); %-#
 
     mask = Cm;       
-    save('mask.mat', 'mask');
+    save(sprintf('swe_vox_mask%s',file_ext), 'mask');
```

Comment:
Just to double check: `file_ext` is `.mat` and not `mat`, right?

Reply:
I can confirm this is '.mat'!